wagtail/images/templates/wagtailimages/bulk_actions/confirm_bulk_add_to_collection.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'wagtailadmin/bulk_actions/confirmation/base.html' %}
+{% load i18n %}
+{% load wagtailimages_tags wagtailadmin_tags %}
+{% block titletag %}{% blocktrans count counter=images|length %}Add 1 image to new collection {% plural %}Add {{ counter }} images to new collection{% endblocktrans %}{% endblock %}
+
+{% block extra_css %}
+    {{ block.super }}
+    {{ form.media.css }}
+{% endblock %}
+
+{% block header %}
+    {% trans "Add images to collection" as add_str %}
+    {% include "wagtailadmin/shared/header.html" with title=add_str icon="doc-full-inverse" %}
+{% endblock header %}
+
+{% block objects_with_access %}
+        {% if images %}
+        <p>
+            {% blocktrans count counter=images|length %}
+            Are you sure you want to add the image to this collection?
```

Comment:
At the moment the question is above the the collection selector so the question doesn't quite make sense. One way to improve this would be to move the collection selector above the question, but then we go down a rabbit hole, so:

suggestion: "Are you sure you want to add the following image to the selected collection?"